best_practices/creating-the-project.rst

```diff
@@ -31,15 +31,15 @@ to create files and execute the following commands:
 .. code-block:: terminal
 
     $ cd projects/
-    $ symfony new blog 3.4
+    $ symfony new --version=3.4 blog
```

Comment:
Perhaps the version option would look better at the end than between the arguments ?

Reply:
I think too but since there were already occurences like this in `setup/symfony_server.rst` I continued.